src/libs/actions/Policy.ts

```diff
@@ -982,6 +985,117 @@ function updateWorkspaceMembersRole(policyID: string, accountIDs: number[], newR
     API.write(WRITE_COMMANDS.UPDATE_WORKSPACE_MEMBERS_ROLE, params, {optimisticData, successData, failureData});
 }
 
+function requestWorkspaceOwnerChange(policyID: string, ownershipChecks?: PolicyOwnershipChangeChecks) {
+    const optimisticData: OnyxUpdate[] = [
```

Comment:
I think we should retrieve `ownershipChecks` from the `POLICY_OWNERSHIP_CHANGE_CHECKS` Onyx key as we described in the doc, but I can't see that implemented here

Reply:
We should always be passing all previously accepted params, i.e. `shouldTransferAmountOwed`, `shouldTransferSubscription`, etc otherwise the API might return an error that was already accepted by the user

Reply:
alright, now it's all clear, I think - I've brought back storing `ownershipChecks` and handle it only in the `requestWorkspaceOwnerChange`